build_scripts/RedHat+Fedora/Linux_RedHat_InstallGuide.md

```diff
@@ -122,7 +124,7 @@ For `Autologin` it is required to append the config option `CONFIG+=rs_autologin
 Example:
 
 ```batch
-qmake CONFIG-=debug CONFIG+=release CONFIG+=rs_use_native_dialog CONFIG+=rs_gui_cmark
+qmake CONFIG+=debug CONFIG+=release CONFIG+=rs_use_native_dialog CONFIG+=rs_gui_cmark
 qmake CONFIG+=rs_jsonapi CONFIG+=rs_webui CONFIG+=rs_autologin
-qmake CONFIG+=rs_deep_channels_index CONFIG += gxsthewire CONFIG += wikipoos
+qmake CONFIG+=rs_deep_channels_index CONFIG +=gxsthewire CONFIG+=wikipoos
```

Comment:
It should be made clear that these options are options (I haven't checked the entire file). Maybe give a "minimal compilation install" line, then describe optional compilation flags such as rs_deep_channels_index and experimental services (gxsthewire, wikipoos, etc)